static/js/ui_util.ts

```diff
@@ -5,10 +5,6 @@ import * as keydown_util from "./keydown_util";
 // Add functions to this that have no non-trivial
 // dependencies other than jQuery.
 
-export function change_tab_to(tabname: string): void {
-    $(`#gear-menu a[href="${CSS.escape(tabname)}"]`).tab("show");
-}
-
```

Comment:
That sounds good, but we need a prep commit to change this line to something like "the message feed visible" instead of checking for this class:

`static/js/message_scroll.js:    if (!$("#message_feed_container").hasClass("active")) {`

Reply:
Yeah, I already added https://github.com/zulip/zulip/pull/23636/commits/a3875fc7aca2c29ceecc8daf83eae031aee273fb for that.